docs/source/kalman_objective.md

```diff
@@ -0,0 +1,481 @@
+# Kalman-type ensemble approaches
+
+How to think about Kalman-type ensemble based data assimilation:
+
+- What does Kalman-type ensemble methods do?
+- What is their ideal objective?
+- How does the objective behave under statistical estimation?
+- How to think about surrogate objectives?
+- What is the modelling setup and what information is known a-priori?
+- List common methods and discuss them in the context of the above.
+
+What is _not_ provided, is an evaluation-comparison between methods.
+Only how we _think_ they will behave, given what we know about statistics and our objective.
+From the contents, it should however be clear how a simulation study could be constructed to compare methods.
+
+
+## Gaussian conditional transport
```

Comment:
The term `Gaussian conditional transport` is new to me, and a search provides no direct hits.
Is this a known term in statistics?

Reply:
It is a shortening of something like "Transporting a joint-Gaussian sample to a (Gaussian-)conditional sample". Perhaps better to spell it out if it is confusing.